docs/src/main/sphinx/develop/client-protocol.rst

```diff
@@ -22,7 +22,7 @@ Overview of query processing
 
 A Trino client request is initiated by an HTTP ``POST`` to the endpoint
 ``/v1/statement``, with a ``POST`` body consisting of the SQL query string.
-The caller may set HTTP header field ``X-Trino-User`` to the username for
+The caller may set the HTTP header field ``X-Trino-User`` to the username for
```

Comment:
Let's change to (make the bracketed a link to the section)
```
The caller may set various [Client request headers]. The headers are
only required on the initial `POST` request and not when following the
``nextUri`` links (as described below).

Reply:
Added with slight mods.